ipv8/attestation/trustchain/block.py

```diff
@@ -504,7 +507,7 @@ def __iter__(self):
         :return: generator to iterate over all properties of this block
         """
         for key, value in self.__dict__.items():
-            if key == 'key' or key == 'serializer' or key == 'crypto' or key == '_transaction':
```

Comment:
Could you bind the set `{'key', 'serializer', 'crypto', '_transaction', '_logger'}` to a module level variable? It's faster.

Reply:
Will do.

Reply:
Done.